packages/web3-providers-http/src/index.ts

```diff
@@ -140,13 +140,13 @@ export default class Web3ProvidersHttp
 
     private async _eth2Request(args: RequestArguments): Promise<AxiosResponse> {
         try {
-            // @ts-ignore tsc doesn't understand httpOptions.method || 'post'
             const response = await this._httpClient[
-                // @ts-ignore tsc will suggest that it can't be some of the methods like 'GET'
-                args?.providerOptions?.AxiosRequestConfig?.method || 'post'
-            ]('', args?.rpcOptions || {}, {
-                ...args?.providerOptions?.axiosConfig,
-            });
+                (args.providerOptions?.httpMethod as 'get' | 'post') || 'get'
+            ](
+                args.method, // URL path
```

Comment:
My suggestion is to remove relying on `axiosConfig` and use `args.method` as the URL path, and `args.params` as `POST` params